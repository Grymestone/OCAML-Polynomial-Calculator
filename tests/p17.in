1a^5 + 5a^7
