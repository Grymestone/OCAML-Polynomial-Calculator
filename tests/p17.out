((1*(a)^5)+(5*(a)^7))
1x^5 + 5x^7